src/framework/shortcuts/internal/shortcutsregister.cpp

```diff
@@ -253,7 +253,14 @@ bool ShortcutsRegister::readFromFile(ShortcutList& shortcuts, const io::path_t&
 
         Shortcut shortcut = readShortcut(reader);
         if (shortcut.isValid()) {
-            shortcuts.push_back(shortcut);
+            if (uiactionsRegister()->action(shortcut.action).isValid()) {
```

Comment:
This code should not be here.
The ShortcutsRegister::readFromFile method should only read data from the file, it should not filter.

Let's just fix the crash. Let's make a hot fix - the default context is any. I hope this doesn't create other critical bugs. At the moment it is a blocking critical bug. Needs to be fixed quickly

And then you'll figure out the problem in another PR

Reply:
@Eism Please have a look. Just pushed it

Reply:
@Eism, the cause of the crash is that these shortcuts in shortcuts.xml don't have a corresponding UiAction in the code:
```cpp
play-next-chord
play-prev-measure
note-breve-TAB
symbols
backspace
viewmode
key-signatures
play-prev-chord
zoom-in-horiz-pre
realtime-advance
next-score
revision
accessibility-dev-show-tree
seek-begin
time-signatures
seek-end
rest-TAB
zoom-in-vert-pre
previous-score
note-longa-TAB
text-word-left
diagnostics-show
play-next-measure
text-word-right
zoom-out-horiz-pre
zoom-out-vert-pre
plugin-creator
startcenter
```

The filtering was dropping any shortcuts that don't have an action, such as these ones above, or others if the user makes a typo in one of the `<key>` tags in shortcuts.xml.

@bkunda is going through this list to see if there are any that we can simply remove from shortcuts.xml because we don't need them anymore, though it still seems like it might be good to filter shortcuts without an action when the file is read in case there are typos. Filtering them will free up the shortcut key so it can be assigned to other things when the user tries to define a new shortcut key in Preferences.